srcIdl/perftest.idl

```diff
@@ -39,9 +39,9 @@ struct TestData_t {
 
 }; //@Extensibility FINAL_EXTENSIBILITY
 
-
 struct TestDataKeyed_t {
-    octet key[KEY_SIZE]; //@key
+    @key
```

Comment:
Check if this compiles for 5.3.0, else, keep the old notation.

Reply:
Yes it does.